taskt/UI/Forms/frmScriptBuilder.cs

```diff
@@ -1646,10 +1646,10 @@ private void SaveToFile(bool saveAs)
                     return;
                 }
 
-                if (!saveFileDialog.FileName.ToString().Contains(_scriptProjectPath))
-                {
-                    Notify("An Error Occured: Attempted to save script outside of project directory");
-                    return;
+                if (!saveFileDialog.FileName.ToString().Contains(_scriptProjectPath))
+                {
+                    Notify("An Error Occured: Attempted to save script outside of project directory");
```

Comment:
Same thoughts as above. If someone wants to do something non-harmful we shouldnt gate it too harshly in my opinion.